app/src/main/java/com/example/capstoneproject/model/api/BitsoResponse.kt

```diff
@@ -0,0 +1,31 @@
+package com.example.capstoneproject.model.api
+
+import com.google.gson.annotations.Expose
+import com.google.gson.annotations.SerializedName
+
+data class BitsoResponse(
+    @SerializedName("success") val success: Boolean,
+    @SerializedName("payload") val payload: List<Currency>
+)
+
+data class Currency(
+    @SerializedName("book") val book: String,
```

Comment:
According to the kotlin coding conventions, they recommend to use camel case names for properties :D https://softwareengineering.stackexchange.com/questions/365339/what-is-wrong-with-magic-strings, how about changing to comply with these conventions? :)

Reply:
Changed! :D